internal/typeinfo/member.go

```diff
@@ -15,7 +15,11 @@ var scannerInterface = reflect.TypeOf((*sql.Scanner)(nil)).Elem()
 // ValueLocator specifies how to locate a value in a SQLair argument type.
 type ValueLocator interface {
 	ArgType() reflect.Type
-	String() string
+	// Desc returns a written description of the ValueLocator for error messages.
+	Desc() string
+	// Repr returns a string representation for the ValueLocator specifying the
+	// type and, if present, member.
+	Repr() string
```

Comment:
Call this `Identifier` with a comment that it uniquely identifies the value in the context of a query.
It should appear above `Desc`.

Reply:
Changed.